kubernetes/nodecollection.go

```diff
@@ -429,7 +432,12 @@ func allowDirectConnect(config KubeAgentConfig, nodes []v1.Node) bool {
 	// if any Fargate nodes are found.
 	for _, n := range nodes {
 		if isFargateNode(n) {
-			log.Infof("Fargate node found in cluster, direct node connection disabled. Learn more about Fargate support: %s", kbURL) //nolint: lll
+			log.Infof("Direct connection to an AWS Fargate node is not possible, so when a Fargate node is " +
+				"detected in the cluster, the agent switches to using kube-proxy for all node connections " +
```

Comment:
I don't think our Proxy connection mode and using kube-proxy are the same thing. I believe when we are in the Proxy mode, we are using kube-api-server as a proxy to collect metrics. Would change "...the agent switches to using kube-proxy for..." to "...the agent's connection mode switches to proxy for..."

Reply:
Fixed